sycl/doc/extensions/DeviceGlobal/SYCL_INTEL_device_global.asciidoc

```diff
@@ -0,0 +1,795 @@
+
+A `device_global` on a given device maintains its state (address of the allocation and data within the allocation) even after the application changes the value of a specialization constant via `handler::set_specialization_constant()`.  Additionally, a `device_global` maintains its state even when it is referenced from a kernel in a different `kernel_bundle`.
+
+[source,c++]
+----
+namespace sycl::ext::oneapi {
+template <typename T, typename PropertyListT = property_list<>>
+class device_global {
+  ...
+----
+
+`device_global` is a class template, parameterized by the type of the underlying allocation _T_, and a list of properties _PropertyListT_. The type of the allocation _T_ also encodes the size of the allocation for potentially multidimensional array types.
+
+_T_ is restricted to types that have a trivial destructor and a trivial default constructor in this revision of the specification (the constructor restriction may be partially relaxed in a future revision). _PropertyListT_ enables properties to be associated with a `device_global`.
+
+Since _T_ is restricted to types with trivial default constructors in this version of the specification, there are no non-default `device_global` constructors, and therefore no initialization values may be specified for the content of a `device_global` allocation on a device.
+
+The allocation of type _T_ is zero-initialized on each device prior to the first access to that `device_global` variable.
+
+`device_global` may only be declared with static storage duration at namespace scope or class scope. If a `device_global` is declared with any other storage duration or scope, the program is ill-formed.
+
+The example below creates two global namespace scope `device_global` objects named `dm1` and `dm2`. `dm1` contains one object of type `MyClass` on each device, and the `device_global` object has external linkage.  `dm2` contains an array of four integers on each device, and the `device_global` object has internal linkage.  In both cases, the `MyClass` and `int[4]` allocations on each device are zero-initialized before any non-initialization accesses occur.
+
+[source,c++]
+----
+using namespace sycl::ext::oneapi;
+
+device_global<MyClass> dm1;
+static device_global<int[4]> dm2;
+
+int main () {
+  sycl::queue Q;
+  Q.submit([&](sycl::handler& h) {
+    h.single_task([=]() {
+      int x = 5;
+      if (dm1.get().flag)
+        x = dm2[0];
+    });
+  });
+}
+----
```

Comment:
This is already covered above. No new details are introduced. Should be removed.

Reply:
Fixed